server.js

```diff
@@ -1,14 +1,9 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import listEndpoints from "express-list-endpoints";
```

Comment:
I really like this snippet of code because it makes it easier to go through your endpoints. Might do that next time!